test/audit/Chambers.t.sol

```diff
@@ -0,0 +1,192 @@
+        rebalancer = new RebalanceWizard();
+        fees = new StreamingFeeWizard();
+
+        god.addWizard(address(issuer));
+        god.addWizard(address(rebalancer));
+        god.addWizard(address(fees));
+
+        address[] memory constituents = new address[](1);
+        constituents[0] = address(USDC);
+
+        uint256[] memory quantities = new uint256[](1);
+        quantities[0] = 1000e6;
+
+        address[] memory wizards = new address[](3);
+        wizards[0] = address(issuer);
+        wizards[1] = address(rebalancer);
+        wizards[2] = address(fees);
+
+        address[] memory managers = new address[](1);
+        managers[0] = manager;
+
+        chamber = Chamber(
+            god.createChamber("Nomoi USDC", "nUSDC", constituents, quantities, wizards, managers)
+        );
+
+        vm.stopPrank();
+
+        vm.startPrank(manager);
+        IStreamingFeeWizard.FeeState memory feeState =
+            IStreamingFeeWizard.FeeState(owner, 1e18, 1e18, block.timestamp);
+
+        fees.enableChamber(chamber, feeState);
+        vm.stopPrank();
+
+        deal(address(USDC), bob, 1000000e6);
+        deal(address(USDC), alice, 1000000e6);
+    }
+
+    function testCollectMultiple() public {
+        vm.startPrank(alice);
+
+        USDC.approve(address(issuer), type(uint256).max);
+        issuer.issue(chamber, 1e18);
+
+        vm.stopPrank();
+
+        vm.startPrank(bob);
+
+        USDC.approve(address(issuer), type(uint256).max);
+        issuer.issue(chamber, 1e18);
+
+        vm.stopPrank();
+
+        console.log("Initial total supply", chamber.totalSupply());
+
+        for (uint256 i; i < 365; ++i) {
+            vm.warp(block.timestamp + 1 days);
+            fees.collectStreamingFee(chamber);
+        }
+
+        vm.warp(block.timestamp + 0.25 days);
+        fees.collectStreamingFee(chamber);
+
+        console.log("total supply", chamber.totalSupply());
+        console.log("time", block.timestamp);
+        console.log("fees received", chamber.balanceOf(owner));
+        console.log("chamber quantity", chamber.getConstituentQuantity(address(USDC)));
+    }
+
+    function testCollectOnce() public {
+        vm.startPrank(alice);
+
+        USDC.approve(address(issuer), type(uint256).max);
+        issuer.issue(chamber, 1e18);
+
+        vm.stopPrank();
+
+        vm.startPrank(bob);
+
+        USDC.approve(address(issuer), type(uint256).max);
+        issuer.issue(chamber, 1e18);
+
+        vm.stopPrank();
+
+        console.log("Initial total supply", chamber.totalSupply());
+
+        vm.warp(block.timestamp + 365.25 days);
+        fees.collectStreamingFee(chamber);
+
+        console.log("total supply", chamber.totalSupply());
+        console.log("time", block.timestamp);
+        console.log("fees received", chamber.balanceOf(owner));
+        console.log("chamber quantity", chamber.getConstituentQuantity(address(USDC)));
+    }
+
+    function testIssueToFakeChamber() public {
```

Comment:
Missing test description, and if its whether SUCCESS or REJECT. Add global sections as well